packages/ui/src/App.tsx

```diff
@@ -13,24 +13,26 @@ import Register from './pages/Register';
 import SecondarySignIn from './pages/SecondarySignIn';
 import SignIn from './pages/SignIn';
 import SocialRegister from './pages/SocialRegister';
-import getSignInExperienceSettings, {
-  parseSignInExperienceSettings,
-} from './utils/sign-in-experience';
+import getSignInExperienceSettings from './utils/sign-in-experience';
 
 import './scss/normalized.scss';
 
 const App = () => {
   const { context, Provider } = usePageContext();
   const { experienceSettings, setLoading, setExperienceSettings } = context;
-  const [isPreview, previewSettings] = usePreview();
+  const [isPreview] = usePreview(context);
 
   useEffect(() => {
+    if (isPreview) {
+      document.body.classList.add('preview');
+
+      return;
+    }
+
```

Comment:
should we remove css class if not preview?

Reply:
also should we use js variable (`styles.xyz`) for these "static" classes? you can import styles like normal scss modules as well.

Reply:
> should we remove css class if not preview?

Since it comes from the query param and in our use case it won't be updated, should be fine?

Reply:
hmm, intend to add it at the body level for all these `static` classes as a global scope classname. Just incase any nested elements need to pick it up.

Reply:
updated